sky/clouds/aws.py

```diff
@@ -791,22 +791,33 @@ def instance_type_exists(self, instance_type):
 
     @classmethod
     def _get_disk_type(cls, disk_tier: resources_utils.DiskTier) -> str:
-        return 'standard' if disk_tier == resources_utils.DiskTier.LOW else 'gp3'
+        if disk_tier == resources_utils.DiskTier.LOW:
+            return 'standard'
+        elif disk_tier == resources_utils.DiskTier.ULTRA:
+            logger.warning(
+                'Using disk_tier=ultra on AWS will utilize io2 Block Express, '
+                'which can lead to significant higher costs. '
+                'For more information, see: https://aws.amazon.com/ebs/pricing.'
+            )
+            return 'io2'
+        return 'gp3'
 
     @classmethod
     def _get_disk_specs(
             cls,
             disk_tier: Optional[resources_utils.DiskTier]) -> Dict[str, Any]:
         tier = cls._translate_disk_tier(disk_tier)
         tier2iops = {
+            resources_utils.DiskTier.ULTRA: 20000,
             resources_utils.DiskTier.HIGH: 7000,
             resources_utils.DiskTier.MEDIUM: 3500,
             resources_utils.DiskTier.LOW: 0,  # only gp3 is required to set iops
         }
         return {
             'disk_tier': cls._get_disk_type(tier),
             'disk_iops': tier2iops[tier],
-            'disk_throughput': tier2iops[tier] // 16,
+            'disk_throughput': tier2iops[tier] // 16 if
+                               tier != resources_utils.DiskTier.ULTRA else None,
```

Comment:
Can io2 configure throughput? If not, it will be great to include a link in the comment for future reference :))

Reply:
No, since throughput is property for `gp3` volume specially. Let me add the reference [link](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ec2-launchtemplate-ebs.html#aws-properties-ec2-launchtemplate-ebs-properties).

Reply:
If that is the case, I guess we could only return disk iops & throughput when needed, and get rid of the `custom_disk_perf` field later in this function. e.g.

```python
{
    'disk_tier': cls._get_disk_type(tier),
    'disk_iops': tier2iops[tier] if cls._get_disk_type(tier) != 'standard' else None,
    'disk_throughput': tier2iops[tier] // 16 if cls._get_disk_type(tier) == 'gp3' else None,
    # delete the 'custom_disk_perf' field
}
```